website/content/en/docs/best-practices/resource-pruning.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Resource pruning"
+linkTitle: "Resource Pruning"
+weight: 3
+description: Recommendations for pruning resources
+---
+
+## Overview
+
+Operators can create [Jobs][jobs] or Pods as part of their normal operation and when those Jobs or Pods
+complete, they can remain on the Kubernetes cluster if not specifically removed.  These resources
+can consume valuable cluster resources like disk storage (e.g. etcd).  The resources here we 
+are discussing are ones that are not tied to a Custom Resource using an ownerReference.
```

Comment:
```suggestion
can consume valuable cluster resources like disk storage (e.g. etcd). These resources
are not tied to a Custom Resource using an `ownerReference`.
```

Reply:
> The resources here we are discussing are ones that are not tied to a Custom Resource using an ownerReference.
This sounds awkward to read. I attempted to rephrase but it might need another rewrite.